README.md

```diff
@@ -34,11 +34,9 @@ On release, automated continuous integration tests run the pipeline on a full-si
 
 ## Pipeline summary
 
-For crispr targeted:
-
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/nf-core/crisprseq/dev/docs/images/crisprseq_targeted_metro_map_dark.png">
-  <img alt="Text changing depending on mode. Light: 'So light!' Dark: 'So dark!'" src="https://raw.githubusercontent.com/nf-core/crisprseq/dev/docs/images/crisprseq_targeted_metro_map.png">
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/nf-core/crisprseq/dev/docs/images/crisprseq_metromap_all.png">
+  <img alt="Text changing depending on mode. Light: 'So light!' Dark: 'So dark!'" src="https://raw.githubusercontent.com/nf-core/crisprseq/dev/docs/images/crisprseq_metromap_all.png">
 </picture>
 
 1. Merge paired-end reads ([`Pear`](https://cme.h-its.org/exelixis/web/software/pear/doc.html))
```

Comment:
We could rephrase the points to merge both subworkflows. Or add teh tube-map as a common figure but keep "For crispr targeted:" from line 37

Reply:
I'm not sure i see what you mean? But i'd remove the old tubemap

Reply:
I agree with removing the tubemap :) I mean we explain the points of each subworkflow with https://github.com/nf-core/crisprseq/blob/master/README.md?plain=1#L58 and https://github.com/nf-core/crisprseq/blob/master/README.md?plain=1#L33
So I wouldn't delete the one for targeted https://github.com/nf-core/crisprseq/pull/128/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L37